charts/base-cluster/values.schema.json

```diff
@@ -1263,6 +1263,102 @@
       },
       "additionalProperties": false
     },
+    "weaveworks": {
+      "type": "object",
+      "description": "Values for the gitops-chart",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "adminUser": {
+          "type": "object",
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "username": {
+              "type": "string"
+            },
+            "createSecret": {
+              "type": "boolean"
+            },
+            "createClusterRole": {
+              "type": "boolean"
+            },
+            "passwordHash": {
+              "type": "string"
+            }
+          }
+        },
+        "ingress": {
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+            "host": {
+              "type": "string"
+            },
+            "tls": {
+              "type": "array"
+            },
+            "className": {
+              "type": "string"
+            }
+          }
+        },
+        "oidcSecret": {
+          "type": "object",
+          "properties": {
+            "additionalKeys": {
+              "type": "object"
+            },
+            "create": {
+              "type": "boolean"
+            }
+          }
```

Comment:
This should be handled like grafana, using `global.authentication`

Reply:
can you take a look at the current state. Haven"t tested it though